src/main/resources/liquibase/changelog-master.yml

```
@@ -1,5 +1,5 @@
 databaseChangeLog:
```

Нужно поменять местами создание лота и бида. Иначе не получится создать связь через внешний ключ

```suggestion
databaseChangeLog:
  - include:
      file: liquibase/changesets/Lot.yml
  - include:
      file: liquibase/changesets/bid.yml
```